db/migrate/20230222115031_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index:true
```

Comment:
- [ ] According to the ERD diagram it is required the user to be named author so I would recommend you change this t.reference to:
`    t.references :author, foreign_key: { to_table: 'users' }`